tests/resource/resource_test_helper.py

```diff
@@ -23,18 +29,19 @@ class DummyModel(Base, ModelBase):
     id = Column('id', INTEGER(), primary_key=True, nullable=False)
     val = Column('val', VARCHAR(length=15), nullable=False)
 
+DummyTuple = namedtuple('DummyTuple', ['id', 'val'])
 
 dummies = [
-    (1, 'one'),
-    (2, 'two'),
-    (3, 'three'),
-    (4, 'four'),
-    (5, 'five'),
-    (6, 'six'),
-    (7, 'seven'),
-    (8, 'eight'),
-    (9, 'nine'),
-    (10, 'ten')
+    DummyTuple(1, 'one'),
+    DummyTuple(2, 'two'),
+    DummyTuple(3, 'three'),
+    DummyTuple(4, 'four'),
+    DummyTuple(5, 'five'),
+    DummyTuple(6, 'six'),
+    DummyTuple(7, 'seven'),
+    DummyTuple(8, 'eight'),
+    DummyTuple(9, 'nine'),
+    DummyTuple(10, 'ten')
 ]

```

Comment:
I'm not sure to have this fixture as a common module in the tests package, but I could think of a better way.